CONTRIBUTORS.yaml

```diff
@@ -14,14 +14,21 @@
 ---
 
 # two dummy entries so that autogenerated tutorials do not fail to build
-contributor1:
-    name: Contributor A (Please add contributors to the CONTRIBUTORS.yaml file)
-    halloffame: "no"
-    joined: 2020-06
-contributor2:
-    name: Contributor B (Please list all contributors of this tutorial in the metadata section)
-    halloffame: "no"
-    joined: 2020-06
```

Comment:
Please re-add these, these should not be removed

Reply:
Yes, we are planning to add more tutorials for other workflows (Retrosynthesis analysis, Pathway analysis, Genetic Design (Gibson, Golden Gate, LCR)). Thank you.

Reply:
Hi @shiltemann , yes I'm in the GTN Gitter. Thank you for your help.